models/prices/polygon/prices_polygon_tokens.sql

```diff
@@ -83,5 +83,6 @@ FROM
     ('woo-wootrade','polygon','WOO',0x1b815d120b3ef02039ee11dc2d33de7aa4a8c603,18),
     ('route-router-protocol','polygon','ROUTE',0x16eccfdbb4ee1a85a33f3a9b21175cd7ae753db4,18),
     ('pla-playdapp','polygon','PLA',0x8765f05adce126d70bcdf1b0a48db573316662eb,18),
-    ('comp-compoundd', 'polygon', 'COMP', 0x8505b9d2254a7ae468c0e9dd10ccea3a837aef5c,18)
+    ('comp-compoundd', 'polygon', 'COMP', 0x8505b9d2254a7ae468c0e9dd10ccea3a837aef5c,18),
+    ('polydoge', 'polygon', 'POLYDOGE', 0x8a953cfe442c5e8855cc6c61b1293fa648bae472,18)
```

Comment:
looks like you're adding an address that already exists in the file, resulting in unique test failure on address:
https://github.com/duneanalytics/spellbook/actions/runs/6881539505/job/18718099296?pr=4803#step:12:25